packages/stream_video/lib/src/call/call.dart

```diff
@@ -1494,15 +1525,11 @@ class Call {
           .contains(settings.video.cameraFacing.value.toLowerCase()),
     );
 
+    // If it's not front or back then take one of the external cameras
     if (defaultVideoInput == null && videoInputs.length > 2) {
-      // If it's not front or back then take one of the external cameras
       defaultVideoInput = videoInputs.last;
```

Comment:
Why is this the last?

Reply:
not sure, looks wrong. Lets address the video input config in separate PR/task.